data/korath/korath news.txt

```diff
@@ -86,4 +86,23 @@ news "kor efret station friendly"
 			"There's a Korath nearby precariously balancing several boxes above its head. You can't tell if it's a street performer or just transporting the crates."
 			"A Kor Efret near your ship is speaking with a worker, gradually increasing its volume until it's doing some strange mix of shouting and hissing. Suddenly, it turns and stalks away with its chest puffed out."
 			"A Korath is studying you intently from a distance. You guess it's never seen a human face to face before."
-			"A local greets you by holding its hands up with its palms facing toward you."
\ No newline at end of file
+			"A local greets you by holding its hands up with its palms facing toward you."
+			
+news "wanderer at kor efret spaceport"
```

Comment:
You should use the new conditional spaceport news (see #5334) to change when this news should show instead of editing the location via an event.

Reply:
Done. I think I had planned to do that and forgotten, thanks!